### PR TITLE
docs(scss): refactor less tabset and tooltip to scss

### PR DESCRIPTION
**Pull request by BY00565233**
## Description

Refactor Tabset and Tooltip Dev Docs from LESS to SCSS

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2084

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: docs/components/tabset/_demo.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.docs-tabset {
```
Comment (100stacks) [inline docs/components/tabset/_demo.scss:1]: Please use `git mv`.

File: docs/components/tooltip/_demo.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.tooltip-appearance {
```
Comment (100stacks) [inline docs/components/tooltip/_demo.scss:1]: Please use `git mv`.

File: docs/styles/demo/tabset-docs.less [hunk 3]
```diff
@@ -1,7 +1,7 @@
-.docs-tabset {
-  #demo-basic {
-    height: 22rem;
-    max-height: 40rem;
-    min-height: 22rem;
-  }
-}
+// .docs-tabset {
```
Comment (100stacks) [inline docs/styles/demo/tabset-docs.less:1]: Please use `git mv`.

File: docs/styles/demo/tooltip-docs.less [hunk 4]
```diff
@@ -1,10 +1,10 @@
-.tooltip-appearance {
-  display: flex;
-  flex-wrap: wrap;
-  z-index: 0;
+// .tooltip-appearance {
```
Comment (100stacks) [inline docs/styles/demo/tooltip-docs.less:1]: Please use `git mv`.

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: docs/components/_index.scss [hunk 5]
```diff
@@ -1,6 +1,8 @@
 /// load styles required to demonstrate each component
 @import "box/index";
 @import "grid/index";
+@import "tabset/demo";
+@import "tooltip/demo";
```
[the remark this answers is not in the document]
Comment (BY00565233) [reply docs/components/_index.scss]: Changes have been done, Please review

File: docs/components/tabset/_demo.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (BY00565233) [reply docs/components/tabset/_demo.scss]: Changes have been done, Please review

File: docs/components/tooltip/_demo.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (BY00565233) [reply docs/components/tooltip/_demo.scss]: Changes have been done, Please review

File: docs/components/tabset/_demo.scss (hunk 1 above)
Comment (BY00565233) [reply docs/components/tabset/_demo.scss]: Changes have been done, Please review

File: docs/components/tooltip/_demo.scss (hunk 2 above)
Comment (BY00565233) [reply docs/components/tooltip/_demo.scss]: Changes have been done, Please review

File: docs/styles/demo/tabset-docs.less (hunk 3 above)
Comment (BY00565233) [reply docs/styles/demo/tabset-docs.less]: Changes have been done, Please review

File: docs/styles/demo/tooltip-docs.less (hunk 4 above)
Comment (BY00565233) [reply docs/styles/demo/tooltip-docs.less]: Changes have been done, Please review

**Review by 100stacks — APPROVED** [review]
Dev LGTM

**Review by lalithkarikelli — APPROVED** [review]
DEV LGTM

**Pull request merged**